src/js/actions/layers.js

```diff
@@ -1377,45 +1382,147 @@ define(function (require, exports) {
     };
 
     /**
+     * Helper function to find the right most artboard 
+     * @param {Immutable.List.of<Layer>} artboards
+     * @return {Layer}
+     */
+
+    var _getRightMost = function (artboards) {
+        var layer = artboards.reduce(function (selectedLayer, currentLayer) {
+                if (currentLayer.bounds.right > selectedLayer.bounds.right) {
+                    return currentLayer;
+                } else {
+                    return selectedLayer;
+                }
+            }, artboards.first());
+        return layer;
+    };
+
+    /**
+     * Helper function to get Bounds from specs (template preset)
+     * @param {string} specs, {Immutable.List.of<Layer>} artboards, {Layer} layerRef
```

Comment:
Separate `@param` declarations needed for each parameter. Also, the type expression on the second parameter is incorrect as above.